bigbluebutton-html5/private/config/settings.yml

```diff
@@ -800,6 +800,7 @@ public:
     showParticipantsOnLogin: true
     showPushLayoutButton: true
     showPushLayoutToggle: true
+    showSessionDetailsOnJoin: false
```

Comment:
Could we default to `true` but add an override to `false` in the ci installation step (iirc our tests were failing due to this)? If we ship as `false` people won't know this exists. If we ship with `true`, people can flip it if/when desired.

Reply:
pushed a commit with the html5 part ([042cecb](https://github.com/bigbluebutton/bigbluebutton/pull/21956/commits/042cecbf8cc12ff0ff7260a4733b0ebe65b7a655)) - @gabriellpr could you change the ci part?